src/lib.rs

```diff
@@ -876,15 +872,11 @@ pub trait Actor {
     /// Default capacity of actor's high-priority inbox unless overridden by `.with_capacity()`.
     const DEFAULT_CAPACITY_HIGH: usize = 5;
 
-    /// The primary function of this trait, allowing an actor to handle incoming messages of a certain type.
-    fn handle(
-        &mut self,
-        context: &mut Self::Context,
-        message: Self::Message,
-    ) -> Result<(), Self::Error>;
-
-    /// The name of the Actor - used only for logging/debugging.
-    fn name() -> &'static str;
+    /// The name of the Actor. Used only for logging/debugging.
```

Comment:
nit: "only" adds no informational value here. We also don't know what the user will use the name for so it's weird to explicitly exclude other usage. 

```suggestion
    /// The name of the Actor. Used for logging/debugging.
```